src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.card {
```

Comment:
Also you can use padding for card, instead of margin-top for image and margin: 0 16px 16px; for card__wrapper(optional)
![image](https://github.com/mate-academy/layout_product-cards/assets/54936044/b85a1327-dbe2-4e3a-9a92-8dfe7c615291)

Reply:
thank you so much! I hope that I understand about `all stars--N modifiers should work as expected` and did it right